10-General.inc.md

```diff
@@ -161,9 +163,9 @@ Note: In theory, it is possible for the [=DRM system=] initialization data to be
 
 ## Signaling presence of encrypted content ## {#CPS-mpd-scheme}
 
-The presence of a `ContentProtection` descriptor with `schemeIdUri="urn:mpeg:dash:mp4protection:2011"` on an adaptation set informs a DASH client that all representations in the adaptation set are encrypted in conformance to Common Encryption ([[!DASH]] 5.8.4.1, 5.8.5.2 and [[!CENC]] 11) and require a [=DRM system=] to provide access.
+The presence of a `ContentProtection` descriptor with `schemeIdUri="urn:mpeg:dash:mp4protection:2011"` on an adaptation set informs a DASH client that all representations in the adaptation set are encrypted in conformance to Common Encryption ([[!DASH]] sections 5.8.4.1 and 5.8.5.2 and [[!CENC]] section 11) and require a [=DRM system=] to provide access.
 
-This descriptor is present for all encrypted content ([[!DASH]] 5.8.4.1). It SHALL be defined on the adaptation set level. The `value` attribute SHALL be either `cenc` or `cbcs`, matching the used [=protection scheme=]. The `cenc:default_KID` attribute SHALL be present and have a value matching the `default_KID` in the `tenc` box. The value SHALL be expressed in lowercase UUID string notation.
+This descriptor is present for all encrypted content ([[!DASH]] section 5.8.4.1). It SHALL be defined on the adaptation set level. The `value` attribute shall be either `cenc` or `cbcs`, matching the used [=protection scheme=] ([[!DASH]] section 5.8.5.2). The `cenc:default_KID` attribute SHALL be present and have a value matching the `default_KID` in the `tenc` box. The value SHALL be expressed in lowercase UUID string notation.
```

Comment:
I think this SHALL should remain in all caps. Why did it become lowercase? Is the intent to remove this requirement?

Reply:
Change to "the value has to be matching the used protection scheme ([[!DASH]] section 5.8.5.2).